JB_README.md

```diff
@@ -0,0 +1,39 @@
+# Extended update concept
+
+This concept is about policy files distribution approach where each client receives from hub only those files that are 
+specific to him. Atop of that, it uses autorun option to execute those. It is about extending existing update process to 
+copy client specific CFEngine policy files from separate repository directory on hub but not located in "masterfiles", since 
+those there will be copied to all clients.
+
+## Description
+
+Client uses it's IP address, hostname and tags file content to enlist subdirectories in hub repository (`$(sys.workdir)/repository/cfengine` 
+or check and adapt `jb_update_policy3.cf` file to your needs). Additional update policy file will look for `$(sys.workdir)/node_tags.dat`
+file where tags can be separated with white space character (\s, \n), coma (,) or semicolon (;). 
+
+Hub repository directory is added to to maintained CFEngine infrastructure, to be shared, with granted access and permissions adjusted (to 0600).
+
+## Schema
```

Comment:
Hi Nick,

sorry for posting/comitting -  still a rookie with my forks of
masterfiles on GitHub.

My blog on that can be found here:

http://juricaborozan.blogspot.com/2016/01/cfengine-and-updatecf-extension.html

and here

http://juricaborozan.blogspot.com/2016/02/cfengine-and-updatecf-extension-2.html


BTW Currently testing CFEngine/Docker possiblities that might be
interesting to commit to library (on blog pages too)

Cheers






On 08.06.16 18:44, Nick Anderson wrote:
> In JB_README.md
> <https://github.com/cfengine/masterfiles/pull/731#discussion_r66292572>:
> 
>> +# Extended update concept
>> +
>> +This concept is about policy files distribution approach where each client receives from hub only those files that are 
>> +specific to him. Atop of that, it uses autorun option to execute those. It is about extending existing update process to 
>> +copy client specific CFEngine policy files from separate repository directory on hub but not located in "masterfiles", since 
>> +those there will be copied to all clients.
>> +
>> +## Description
>> +
>> +Client uses it's IP address, hostname and tags file content to enlist subdirectories in hub repository (`$(sys.workdir)/repository/cfengine` 
>> +or check and adapt `jb_update_policy3.cf` file to your needs). Additional update policy file will look for `$(sys.workdir)/node_tags.dat`
>> +file where tags can be separated with white space character (\s, \n), coma (,) or semicolon (;). 
>> +
>> +Hub repository directory is added to to maintained CFEngine infrastructure, to be shared, with granted access and permissions adjusted (to 0600).
>> +
>> +## Schema
> 
> This is interesting. Would you be interested in writing a guest blog
> post about this?
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/cfengine/masterfiles/pull/731/files/0fa6c41b875a7329d85122a93e4e5b9c770796a4#r66292572>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFPh_rmlf0ElEs--0wPOlcRdTLg2rsJyks5qJvFegaJpZM4IxKD2>.
>